src/benchmark/trulens/benchmark/benchmark_frameworks/experiments/rag_triad_baselines.ipynb

```diff
@@ -23,241 +23,29 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.        write_results(
</pre><p>It'd be better to show the results with trulens-native apis e.g. get_leaderboard rather than writing to csv</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1559/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>